qa/workunits/rbd/nvmeof_fio_test.sh

```diff
@@ -3,8 +3,44 @@
 sudo yum -y install fio
 sudo yum -y install sysstat
 
+namespace_range_start=
+namespace_range_end=
+rbd_iostat=false
+
+while [[ $# -gt 0 ]]; do
+    case "$1" in
+        --start_ns)
+            namespace_range_start=$2
+            shift 2
+            ;;
+        --end_ns)
+            namespace_range_end=$2
+            shift 2
+            ;;
+        --rbd_iostat)
+            rbd_iostat=true
+            shift
+            ;;
+        *)
+            exit 100	# Internal error
+            ;;
+    esac
+done
+
 fio_file=$(mktemp -t nvmeof-fio-XXXX)
-drives_list=$(sudo nvme list --output-format=json | jq -r '.Devices | .[] | select(.ModelNumber == "SPDK bdev Controller") | .DevicePath')
+all_drives_list=$(sudo nvme list --output-format=json | 
+    jq -r '.Devices | sort_by(.NameSpace) | .[] | select(.ModelNumber == "SPDK bdev Controller") | .DevicePath')
+
+# When the script is passed --start_ns and --end_ns (example: `nvmeof_fio_test.sh --start_ns 1 --end_ns 3`), 
+# then fio runs on namespaces only in the defined range (which is 1 to 3 here). 
+# So if `nvme list` has 5 namespaces with "SPDK Controller", then fio will 
+# run on first 3 namespaces here.
+if [ "$namespace_range_start" ] || [ "$namespace_range_end" ]; then
+    selected_drives=$(echo "${all_drives_list[@]}" | sed -n "${namespace_range_start},${namespace_range_end}p")
+else
+    selected_drives="${all_drives_list[@]}"
+fi
+
 
 RUNTIME=${RUNTIME:-600}
 # IOSTAT_INTERVAL=10
```

Comment:
stray comment

Reply:
This is the only env variable without a default value in the script, that's why I added it as a comment. I'll remove it if it looks out of place.

Reply:
It's not so much that it looks out place, it's that one can't tell why it's there when reviewing.  I'd suggest replacing it with something like
> Set IOSTAT_INTERVAL in job yaml

or just removing it.

Reply:
I see, I've removed the comment, thank you!